app/src/behaviors/behavior_hold_tap.c

```diff
@@ -360,7 +438,7 @@ static int release_binding(struct active_hold_tap *hold_tap) {
 }
 
 static void decide_hold_tap(struct active_hold_tap *hold_tap,
-                            enum decision_moment decision_moment) {
+                            struct decision_trigger_event *decision_trigger_event) {
```

Comment:
I think I prefer passing three parameters into this method instead of the struct. _If_ we take a struct as input, I think we should include the `*hold_tap` in it too.